src/modules/users/users.controller.ts

```diff
@@ -1,3 +1,5 @@
+import * as Sentry from '@sentry/node';
```

Comment:
Do we need to import sentry in all other controllers?

Reply:
I don't think so; only where we need to use its API directly; for example setting tags, configuring the user, or calling sentry explicitly.